### PR TITLE
use thanos to access old data

### PR DESCRIPTION
**Pull request by tomhalson**

### CODE REVIEW

File: helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json [hunk 1]
```diff
@@ -382,20 +382,7 @@
           },
           "mappings": []
         },
-        "overrides": [
```
Comment (JamesBoobierMOJ) [inline helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json:385]: This should be retained

File: helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json [hunk 2]
```diff
@@ -602,8 +590,8 @@
       {
         "current": {
           "selected": false,
-          "text": "hmpps-resettlement-passport-dev",
-          "value": "hmpps-resettlement-passport-dev"
+          "text": "hmpps-resettlement-passport-prod",
+          "value": "hmpps-resettlement-passport-prod"
```
Comment (JamesBoobierMOJ) [inline helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json:594]: Default probably should be dev

Comment (tomhalson) [reply helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json:594]: not sure about this, the data we need to report on is prod, and we want to open this up to the wider team

Comment (JamesBoobierMOJ) [reply helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json:607]: Good point - let's point it to prod then.

**Review by JamesBoobierMOJ — APPROVED** [review]

**Pull request merged**